### PR TITLE
Added steps to install plugin

### PR DESCRIPTION
**Pull request by freakomonk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,14 @@
 # goto-audio
 Goto the tab that is playing audio. Exclusive for Chrome
+
+Steps to install goto-audio
```
Comment (fhalde) [inline README.md]: prefix this with # so that it becomes a header. try putting a horizontal line between the description and this header! Thanks

**Comment by fhalde** [comment]
Seven commits !! There you go, a new lesson for the day. Squash them into one commit !
@skarthikeya .. cant do using GitHub unfortunately

**Comment by freakomonk** [comment]
Yeah. Working on it :|

**Comment by fhalde** [comment]
You need not close this PR. Same can be used

**Comment by fhalde** [comment]
still wrong! why is that second file there!

**Comment by fhalde** [comment]
let me guess .. `git add -A` .. stop using it !!!!!

**Comment by freakomonk** [comment]
Yeah. I did a pull and then solved the conflicted files. Now PR should be okay. Please check

**Pull request merged**